packages/strapi-utils/lib/models.js

```diff
@@ -454,7 +454,11 @@ module.exports = {
       let formattedValue;
 
       try {
-        formattedValue = !_.isNaN(_.toNumber(value)) ? _.toNumber(value) : value;
+        formattedValue = _.isObject(value) // Do not format object or array values
```

Comment:
Can someone enlighten me a little more about this change, because it's causing a lot of problems.
as in example if the `_sort` is not defined it will use the default value which is an empty string and `_.toNumber(value) = 0` which means that here https://github.com/strapi/strapi/pull/1345/files#diff-bf3a6e5f9eb97177916c46670dc7046eR469 it'll break saying: `TypeError: formattedValue.split is not a function or its return value is not iterable` we need to find a better way to re-write this logic. Can you please explain to me what are we trying to resolve here.